index.js

```diff
@@ -1,8 +1,21 @@
 /*!
- * Copyright (c) 2012-2024 Digital Bazaar, Inc.
+ * Copyright 2024 Digital Bazaar, Inc.
  *
- * SPDX-License-Identifier: BSD-3-Clause
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Is there an acceptable abbreviated form for this or must it be so verbose in every file?

Reply:
You mean the stuff above the `SPDX-License-Identifier` line?

That's from https://www.apache.org/licenses/LICENSE-2.0#apply

So, yes, it's required.